photon-core/src/main/java/org/photonvision/vision/frame/consumer/FileSaveFrameConsumer.java

```diff
@@ -102,4 +102,8 @@ public void updateCameraNickname(String newCameraNickname) {
         this.subTable.getEntry(ntEntryName).setInteger(imgSaveCountInternal);
         this.entry = subTable.getIntegerTopic(ntEntryName).getEntry(-1); // Default negative
     }
+
+    public void overrideTakeSnapshot() {
+        entry.set(entry.get() + 1);
```

Comment:
lmao cursed old code. Worth dropping some comments on what it's doin

Reply:
I just refactored it, it makes a lot more sense with better variable names